kazoo/recipe/lock.py

```diff
@@ -555,6 +566,10 @@ def _inner_release(self):
         self.is_acquired = False
         return True
 
+    def break_lock(self):
```

Comment:
Just wondering about this in the semaphore class, since the lock usage inside of this is somewhat different than the lock usage in the main lock class, maybe needs more advanced breaking logic?